pyproject.toml

```diff
@@ -4,9 +4,22 @@ build-backend = "maturin"
 
 [project]
 name = "fastexcel"
+description = "A fast excel file reader for Python, written in Rust"
+readme = "README.md"
+license = { file = "LICENSE" }
 requires-python = ">=3.8"
 classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
     "Programming Language :: Rust",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.9",
```

Comment:
We can add 3.8 here as well

Reply:
https://github.com/ToucanToco/fastexcel/pull/290/commits/84bc358d8935ad82611ff63c3b63e28501c5fc80